index.bs

```diff
@@ -494,9 +494,12 @@ with the following additional codes:
 <pre class="cddl local-cddl">
 ErrorCode = "invalid argument" /
             "invalid session id" /
+            "move target out of bounds" /
```

Comment:
Good catch given that this is part of the classic spec the performActions command hooks into.